packages/ra-language-french/index.js

```diff
@@ -62,6 +71,7 @@ module.exports = {
             updated: 'Élément mis à jour',
             created: 'Élément créé',
             deleted: 'Élément supprimé',
+            bad_item: 'Élément incorrect',
```

Comment:
s/incorrect/inconnu/

Reply:
btw, I think this one is missing in the english package too

Reply:
Nope, it isn't https://github.com/marmelab/admin-on-rest/blob/next/packages/ra-language-english/index.js#L72